vitest.config.ts

```diff
@@ -128,32 +128,16 @@ export const sharedConfig = defineConfig({
   },
 });
 
-export default defineConfig(async ({ mode }) => {
-  // Resolve the Vitest configuration for PrairieLearn with the given mode.
-  // By default, the configuration is not resolved with the mode.
-  const { vitestConfig: prairielearnTestConfig } = await resolveConfig({
-    mode,
-    config: 'vitest.config.ts',
-    root: 'apps/prairielearn',
-  });
-  return mergeConfig(
-    sharedConfig,
-    defineConfig({
-      test: {
-        projects: [
-          'packages/*',
-          'apps/grader-host',
-          'apps/workspace-host',
-          {
-            test: prairielearnTestConfig,
-          },
-        ],
-        coverage: {
-          all: true,
-          reporter: ['html', 'text-summary', 'cobertura'],
-          include: ['{apps,packages}/*/src/**'],
-        },
+export default mergeConfig(
```

Comment:
This change is from #12235 -- resolving the config twice is bad practice I realized. This removes that bad practice, but you can no longer pass in the 'mode' flag at the root level. Since we just use 'mode' for CI, and CI never runs from the root (only from apps/prairielearn), I think this is fine.